test/check-cli

```diff
@@ -102,6 +102,21 @@ class TestTar(composertest.ComposerTestCase):
             # Run the test, on the booted image
             self.runImageTest("/tests/cli/test_boot_tar_kickstart.sh")
 
+class TestPylint(composertest.ComposerTestCase):
+    def test_ci(self):
+
+        # Create an empty folder for the upload bellow
+        self.execute(["mkdir /lorax"])
```

Comment:
This should be ["mkdir", "/lorax"]

Reply:
Thanks, I changed it and squashed with the first commit.